project/Dependencies.scala

```diff
@@ -85,7 +84,7 @@ object Dependencies {
   val workbenchLibsHash = "46d1df6"
 
   val workbenchModelV  = s"0.15-${workbenchLibsHash}"
-  val workbenchGoogleV = s"0.22-${workbenchLibsHash}"
+  val workbenchGoogleV = s"0.23-6466499a-SNAP"
```

Comment:
Will update to non-SNAP once https://github.com/broadinstitute/workbench-libs/pull/1040 is merged